hangouts.cpp

```diff
@@ -63,35 +63,73 @@ SimpleHttps	*https;
 	pair<string, string> header = make_pair("Content-type", "application/json");
 	headers.push_back(header);
 
+	if (m_url.empty())
```

Comment:
You should also trim it first, If I put `space` in the field, then in syslog-

`Sep  3 16:47:08 foglamp-ThinkPad-E450 FogLAMP Notifications[16987]: ERROR: Exception while sending notification to hangouts webhook  : basic_string::substr: __pos (which is 2) > this->size() (which is 1)`

Reply:
Please create another JIRA

This fix for failure need to be merged to develop